libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -130,11 +130,14 @@ public static String getJourneyEvent(JourneyRequest request)
 
     public static String getJourneyParameter(JourneyRequest request, String key)
             throws HttpResponseExceptionWithErrorBody {
-        List<NameValuePair> queryParams = new URIBuilder(request.getJourneyUri()).getQueryParams();
-        return queryParams.stream()
-                .filter(query -> Objects.equals(query.getName(), key))
+        return Optional.ofNullable(request.getJourneyUri().getQuery())
```

Comment:
this new logic is due to losing an apache lib - must have been a transative dep of the v1 sdk. Implemented this rather than pulling a library back in.